packages/core/src/amazonq/webview/ui/main.ts

```diff
@@ -642,8 +645,24 @@ export class WebviewUIHandler {
         })
 
         this.mynahUIProps = {
-            onReady: this.connector.uiReady,
+            onReady: () => {
+                // the legacy event flow adds events listeners to the window, we want to avoid these in the lsp flow, since those
+                // are handled by the flare chat-client
+                if (hybridChat && this.connector) {
+                    this.connector.isUIReady = true
+                    postMessage({
+                        command: 'ui-is-ready',
+                    })
+                    return
+                }
+                this.connector?.uiReady()
+            },
             onTabAdd: (tabID: string) => {
+                if (hybridChat) {
+                    // hybrid chat doesn't support the welcome page
```

Comment:
Does this mean its currently not shown? Are there plans to re-add it or is it something we can drop entirely?

Reply:
AFAIK for agentic chat it's been dropped (or at least de-prioritized)